best-practice/index.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * This script generates the best-practices.md file from the definitions in the definitions.ts file.
+ *
+ * Add new best practices to the definitions.ts file.
+ */
+
+import * as path from "https://deno.land/std/path/mod.ts";
+import { markdownTable } from "npm:markdown-table";
+import { AllBestPractices } from "./definitions.ts";
+
+const markdownFile = path.join(
+  path.dirname(path.fromFileUrl(import.meta.url)),
+  "..",
+  "best-practices.md",
+);
+
+const file = await Deno.readTextFile(markdownFile);
+
+const startMark = "<!-- csvstart -->";
+const endMark = "<!-- csvend -->";
```

Comment:
Might be a little confusing if we're not using CSV…

Reply:
🤦🏽  will update.